README.md

```diff
@@ -32,7 +32,7 @@ See [examples/docker](examples/docker)
 ### MacPorts
```

Comment:
Do you want to update this line to just say MacOS?

Reply:
I agree, it's better.